leaf-ui/scripts/main.js

```diff
@@ -257,10 +257,6 @@ $('#analysis-btn').on('click', function(){
 //Switch to modeling mode
 $('#model-cur-btn').on('click', function(){
 	switchToModellingMode(false);
-});
-
-$('#model-init-btn').on('click', function(){
```

Comment:
This is a dead code. In the current project we don't have any html element with this id.